science/ViennaRNA/Portfile

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           compilers 1.0
+name                ViennaRNA
+version             2.4.14
+categories          science
+platforms           darwin
+maintainers         @GiovanniBussi openmaintainer
+description         C Library and programs for the prediction and comparison\
+                    of RNA secondary structures.
+long_description    The Vienna RNA Package consists of a C code library and\
+                    several stand-alone programs for the prediction and\
+                    comparison of RNA secondary structures.
+homepage            https://www.tbi.univie.ac.at/RNA/
+master_sites        http://www.tbi.univie.ac.at/RNA/download/sourcecode/2_4_x/ViennaRNA-${version}.tar.gz
+# http://www.tbi.univie.ac.at/~ivo/RNA/COPYING.html
+license             restrictive
+
+distfiles           ViennaRNA-${version}.tar.gz
+
+checksums           rmd160  8c415cfd97d09c12abd99c319313971badb17aeb \
+                    sha256  ba9cfc8a48e457fc891628f3229a3924de31714460dc4a4dec081868f802cc28 \
+                    size    32814278
+
+compilers.choose    cc cxx
+compilers.setup
```

Comment:
Why does this port need the compilers portgroup?

Reply:
I removed it, indeed it is not needed for the port to compile correctly.